charts/camunda-platform/test/unit/connectors/deployment_test.go

```diff
@@ -271,7 +271,7 @@ func (s *deploymentTemplateTest) TestContainerSetContainerCommand() {
 	options := &helm.Options{
 		SetValues: map[string]string{
 			"connectors.enabled": "true",
-			"connectors.command": "[printenv]",
+			"connectors.command": "{printenv}",
```

Comment:
This is not correct.
The command is a list, not a dict.

Reply:
Let me check again, but I applied it to the set values as a list according to the [Helm docs](https://helm.sh/docs/intro/using_helm/#the-format-and-limitations-of---set) because it failed before.

Reply:
Correct me if I am wrong, but I think you want to test the Helm Options like you would set the options - and [the Helm docs states](https://helm.sh/docs/intro/using_helm/#the-format-and-limitations-of---set) for lists:
> Lists can be expressed by enclosing values in { and }

Reply:
@ttretau you are actually correct.
I totally forgot about it, it's confusing in the testing code.

For the same reason, we don't use that format :grinning: 
So let's follow the same pattern in the other tests and make it as:

```
"connectors.command[0]": "printenv",
```

Reply:
@aabouzaid Right, that looks possibly clearer now..